HomeWork/src/main/java/multithreading/Main.java

```diff
@@ -0,0 +1,17 @@
+package multithreading;
+
+import java.util.Arrays;
+import java.util.HashMap;
+
+public class Main {
+    public static void main(String[] args) throws InterruptedException {
+
+        ValueCalculator valueCalculator = new ValueCalculator();
+        Thread thread1 = new Thread(valueCalculator, " Thread 1");
+        Thread thread2 = new Thread(valueCalculator, " Thread 2");
```

Comment:
These threads should be started in method `run` not here. Please check the task description carefully.